ui/components/component-library/select-button/select-button.stories.tsx

```diff
@@ -0,0 +1,252 @@
+import { StoryFn, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  AvatarBase,
+  AvatarBaseSize,
+  AvatarAccount,
+  AvatarAccountSize,
+  Box,
+  SelectWrapper,
+  SelectOption,
+  Text,
+} from '..';
+import {
+  Display,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import README from './README.mdx';
+
+import { SelectButtonSize } from './select-button.types';
+import { SelectButton } from '.';
+
+export default {
+  title: 'Components/ComponentLibrary/SelectButton',
+  component: SelectButton,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {},
```

Comment:
Add argTypes here? LPT: Used chatGPT to convert types to argTypes
```suggestion
  argTypes: {
  className: { control: 'text' },
    size: { control: { type: 'select', options: [SelectButtonSize.Sm, SelectButtonSize.Md, SelectButtonSize.Lg] } },
    placeholder: { control: 'object' },
    value: { control: 'object' },
    defaultValue: { control: 'object' },
    isBlock: { control: 'boolean' },
    isDanger: { control: 'boolean' },
    isDisabled: { control: 'boolean' },
    disabled: { control: 'boolean' },
    label: { control: 'text' },
    labelProps: { control: 'object' },
    description: { control: 'text' },
    descriptionProps: { control: 'object' },
    startAccessory: { control: 'text' },
    endAccessory: { control: 'text' },
    caretIconProps: { control: 'object' },
  },
```

Reply:
I started adding them back because storybook used to break for props that didn't have argTypes defined. Even for things like strings which seems like a pretty straightforward. But I just tested it and it doesn't seem to be an issue any more so you can disregard this comment.